### PR TITLE
build: Remove travis references

### LINKED ISSUE #2154: Build Status icons still link to Travis
Can these be changed to show Github Actions, or should they just be removed?

image

### PR DESCRIPTION
**Pull request by barton2526**
before_install.sh might be able to be removed entirely, see (https://github.com/bitcoin/bitcoin/pull/20691/commits/09d105ef0f8b4b06bf248721a1209c9e16e9db75).

### CODE REVIEW

**Comment by barton2526** [comment]
@jamescowens if you want this to close #2154 I need to know whether we want to replace the icons or remove them entirely

**Comment by jamescowens** [comment]
Replace

**Comment by barton2526** [comment]
Here's what it looks like with this PR
image

File: ci/lint/06_script.sh [hunk 1]
```diff
@@ -6,8 +6,8 @@
 
 export LC_ALL=C
 
-if [ "$TRAVIS_EVENT_TYPE" = "pull_request" ]; then
-  test/lint/commit-script-check.sh $TRAVIS_COMMIT_RANGE
+if [ "$CI_EVENT_TYPE" = "pull_request" ]; then
```
Comment (div72) [inline ci/lint/06_script.sh]:
```suggestion
if [ "$EVENT_TYPE" = "pull_request" ]; then
```
or (remove the env this env var in the workflow)
```suggestion
if [ "$GITHUB_EVENT_TYPE" = "pull_request" ]; then
```
If you want to preserve the `CI_EVENT_TYPE`, you could edit workflow file to change the env name.

**Review by div72 — CHANGES_REQUESTED** [review]
The before_install script contains the folds which are supported in GitHub actions.

File: ci/lint/06_script.sh [hunk 2]
```diff
@@ -6,8 +6,8 @@
 
 export LC_ALL=C
 
-if [ "$TRAVIS_EVENT_TYPE" = "pull_request" ]; then
-  test/lint/commit-script-check.sh $TRAVIS_COMMIT_RANGE
+if [ "$GITHUB_EVENT_TYPE" = "pull_request" ]; then
+  test/lint/commit-script-check.sh $CI_COMMIT_RANGE
 fi
```
Comment (div72) [inline ci/lint/06_script.sh]:
```suggestion
```
Forgot that it's already handled underneath.

**Comment by barton2526** [comment]
gonna need to be squashed lol

**Comment by jamescowens** [comment]
@div72 can you check this one last time and make sure you are satisfied?

**Review by div72 — APPROVED** [review]
@jamescowens I was waiting on barton to squash.

**Comment by jamescowens** [comment]
I am squashing on merge.

**Pull request merged**